spec/lib/crawler/coordinator_spec.rb

```diff
@@ -38,6 +38,7 @@
   let(:sink) { Crawler::OutputSink::Mock.new(crawl_config) }
   let(:crawl_queue) { Crawler::Data::UrlQueue::MemoryOnly.new(crawl_config) }
   let(:seen_urls) { Crawler::Data::SeenUrls.new }
+  let(:crawl_result) { FactoryBot.build(:html_crawl_result, content: '<p>BOO!</p>') }
```

Comment:
👻 !

Reply:
Sometimes I wonder if you leave things like this in here just to see if we're actually reading/reviewing closely. :)

Reply:
I'm definitely not that sly, they're just easter eggs so I can laugh when I have to fix a test in a year :)